packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailOverview.tsx

```diff
@@ -0,0 +1,87 @@
+import { CircleProgress } from '@linode/ui';
+import { Grid, Typography, useTheme } from '@mui/material';
+import React from 'react';
+
+import { useCloudPulseServiceTypes } from 'src/queries/cloudpulse/services';
+
+import { convertStringToCamelCasesWithSpaces } from '../../Utils/utils';
+import { severityMap } from '../constants';
+import { formatTimestamp, getServiceTypeLabel } from '../Utils/utils';
+import { AlertDetailRow } from './AlertDetailRow';
+
+import type { Alert } from '@linode/api-v4';
+
+interface OverviewProps {
+  /*
+   * The alert for which the criteria is displayed
+   */
+  alert: Alert;
+}
+export const AlertDetailOverview = (props: OverviewProps) => {
+  const { alert } = props;
+
+  const {
+    created_by: createdBy,
+    description,
+    label,
+    service_type: serviceType,
+    severity,
+    status,
+    type,
+    updated,
+  } = alert;
+
+  const { data: serviceTypes, isFetching } = useCloudPulseServiceTypes(true);
+
+  const theme = useTheme();
+
+  const statusColorMap = {
+    disabled: theme.color.grey1,
+    enabled: theme.color.green,
+    failed: theme.color.red,
+    provisioning: theme.color.orange,
+  };
+
+  if (isFetching) {
+    return <CircleProgress />;
+  }
+
+  return (
+    <React.Fragment>
+      <Typography
+        fontSize={theme.spacing(2.25)}
+        gutterBottom
```

Comment:
Do we need this? as marginBottom is anyway defining the space right?

Reply:
not needed